bats/tags.bats

```diff
@@ -197,6 +185,6 @@ SQL
     dolt checkout master
     dolt merge branch1
     run dolt merge branch2
-    [ $status -ne 0 ]
+    [ $status -eq 0 ]
```

Comment:
Why does this now work. I added two different columns with the same tag on the different branches and now we can merge it?

Reply:
If the column has the same name, tag and type then we don't interpret it as a merge conflict. 

It sounds like this test was meant to test for are error if `c0` and `c2` with the same tag and type are merged. I'll add another test that defines the behavior for this case (same name, tag, and type created on separate branches). Do we want to allow merging of identical columns created on separate branches?

In the case of `c0` and `c2` we still will not error exit. In order to hide the details of tags, the current behavior does an automatic rebase for tag conflicts during `merge`. Is this reasonable to automatically resolve tag conflicts?

Reply:
Column c0's tag is now being automatically rebased to a new unique tag. 
Because tag conflicts can now happen not only within a table, but across tables, I implemented automatic tag conflict resolution by choosing new tags for the conflicts and rebasing the merge branch. Is this reasonable?

Reply:
This strikes me as taking too much control away from the user. I think this is cool for automatically generated tags. But if a user goes to the trouble of defining a tag I don't think we should automatically resolve the conflict. Do we distinguish between user-supplied and auto-generated tags? 

And if I had to choose between conflict and no conflict, I would choose a merge conflict I think.

Reply:
no longer automatically rebasing